vnext/Microsoft.ReactNative/Modules/AccessibilityInfoModule.cpp

```diff
@@ -71,4 +71,10 @@ void AccessibilityInfo::announceForAccessibility(std::string announcement) noexc
   });
 }
 
+void AccessibilityInfo::getRecommendedTimeoutMillis(
+    double mSec,
+    std::function<void(React::JSValue const &)> const &onSuccess) noexcept {
+  mSec = 0;
```

Comment:
As Alex said, I think the 'returning' would be via calling the onSuccess callback, and passing it mSec as an argument (just like the Android implementation does). Otherwise, you're just assigning to a local variable that gets discarded, and the result doesn't reach JS.

Reply:
Oh I see got it